ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/TestKsqlRestApp.java

```diff
@@ -108,6 +111,10 @@
   private KsqlExecutionContext ksqlEngine;
   private KsqlRestApplication ksqlRestApplication;
 
+  static {
+    // Increase the default - it's low (100)
+    System.setProperty("sun.net.maxDatagramSockets", "1024");
```

Comment:
Having trouble understanding what this property means... how come it needs to be increased here?

Reply:
When doing DNS lookups, Netty uses it's own resolver that uses UDP, and it caches resolvers per event loop. When we start the api server we create 2 * number of cores * number of listeners event loops. Also in some tests we start multiple api servers. This means the number of udp sockets required can be > the OS default, which is 100, which will cause tests to fail.

Reply:
Thanks for the context! Very helpful.